plugins/messagelayer/plugin.go

```diff
@@ -204,7 +204,7 @@ func newTangle(tangleDeps tangledeps) *tangle.Tangle {
 	)
 
 	tangleInstance.Scheduler = tangle.NewScheduler(tangleInstance)
-	tangleInstance.WeightProvider = tangle.NewCManaWeightProvider(GetCMana, tangleInstance.TimeManager.Time, tangleDeps.Storage)
+	tangleInstance.WeightProvider = tangle.NewCManaWeightProvider(GetCMana, tangleInstance.TimeManager.RATT, tangleDeps.Storage)
```

Comment:
Now the TT doesn't stop anymore and the network should not halt. However, in the weightprovider we still have 30min threshold. Should we also decrease that?